### DISCUSSION TITLE
Wiring Setup

### DISCUSSION THREAD

**Discussion by ndrinta** [FozzieUK/FoxESS-Canbus-Protocol#4] [Q&A] [answered]
I have an H1 (Gen1), I was wondering if the available documentation is correct?

![image](https://github.com/user-attachments/assets/68a7bb39-ad57-4131-bed6-b1151a123e41)

- Pin 2: GND
- Pin 4: CAN_L
- Pin 5: CAN_H

**Comment by FozzieUK** [comment, marked as answer]
Yes that's the correct pinouts for the H1 series.

The easiest way to connect to the inverter is to remove the cover with the 4 small screws and 2 cable glands which gives access to the Com / CAN debug ports - these carry exactly the same signals for the BMS so you can leave that connected and simply connect here.

![111](https://github.com/user-attachments/assets/beaea280-c2f0-4e4a-97e0-922d24e3bc8c)
